docs/specifications/tests/DNSSEC-TP/dnssec15.md

```diff
@@ -0,0 +1,188 @@
+
+6.  Repeat the following steps for each name server IP address in *NS IP*:
+
+    1. Send the CDS query over UDP to the name server IP address.
+       1. If no DNS response is returned, then go to next name server IP.
+       2. Else, if AA bit is not set in the DNS response, then go to next 
+          name server IP.
+       3. Else, if the RCODE in the DNS response is not *NOERROR*, then go to
+          next name server IP.
+       4. Else, if the DNS response contains no CDS record in the
+          answer section, then add the name server IP and an empty RRset to
+          the *CDS RRsets* set.
+       5. Else, add the name server IP and the CDS RRset from the answer
+          section to the *CDS RRsets* set.
+    2. Send the CDNSKEY query over UDP to the name server IP address.
+       1. If no DNS response is returned, then go to next name server IP.
+       2. Else, if AA bit is not set in the DNS response, then go to next
+          name server IP.
+       3. Else, if the RCODE in the DNS response is not *NOERROR*, then go to
+          next name server IP.
+       4. Else, if the DNS response contains no CDNSKEY record in the
+          answer section, then add the name server IP and an empty RRset to
+          the *CDNSKEY RRsets* set.
+       5. Else, add the name server IP and the CDNSKEY RRset from the answer
+          section to the *CDNSKEY RRsets* set.
+    3. Go to next name server IP.
+
+7.  If the *CDS RRsets* set and the *CDNSKEY RRsets* set are empty
+    then output *[DS15_NO_CDS_CDNSKEY]* and terminate this
+    test case.
+
+8.  Repeat the following steps for each name server IP address in *NS IP*:
+
+    1. If the name server IP address has a non-empty RRset in the
+       *CDS RRsets* set, but an empty RRset in the *CDNSKEY RRsets*
+       set, then add the name server IP address to *Has CDS No CDNSKEY*.
+    2. If the name server IP address has a non-empty RRset in the
+       *CDNSKEY RRsets* set, but an empty RRset in the *CDS RRsets*
+       set, then add the name server IP address to *Has CDNSKEY No CDS*.
+    3. If the name server IP address has a non-empty RRset in both
+       sets, *CDNSKEY RRsets* and *CDS RRsets*, then add the name
+       server IP address to *Has CDS And CDNSKEY*.
+    4. Go to next name server IP.
+
+8.  If the *Has CDS No CDNSKEY* set is non-empty then output
+    *[DS15_HAS_CDS_NO_CDNSKEY]* with the name server IP addresses from
+    the set.
+
+9.  If the *Has CDNSKEY No CDS* set is non-empty then output
+    *[DS15_HAS_CDNSKEY_NO_CDS]* with the name server IP addresses from
+    the set.
+
+9.  If the *Has CDS And CDNSKEY* set is non-empty then output
+    *[DS15_HAS_CDS_AND_CDNSKEY]* with the name server IP addresses from
+    the set.
+
+11. If not all CDS RRsets in the *CDS RRsets* set are identical, where
+    a non-empty RRset is not considered to be identical to an empty
+    RRset, then output *[DS15_INCONSISTENT_CDS]*.
```

Comment:
`not considered to be identical to` -> `considered to be different from`

There's another one like this below.

Reply:
Will fix.